tests/models/test_system.py

```diff
@@ -80,10 +80,7 @@ def test_system_str_jobs(self):
         job.add_build(build)
         self.system.add_job(job)
         output = str(self.system)
```

Comment:
We should add the verbosity argument True here, this tests was added to check that the full job and build information is printed correctly